forage-android/src/main/java/com/joinforage/forage/android/core/element/state/PanElementStateManager.kt

```diff
@@ -56,27 +68,82 @@ class PanElementStateManager(state: ElementState) : ElementStateManager(state) {
             null
         }
     }
+}
+
+open class WhitelistedCards(
+    private val specialChar: String,
+    private val prefixLength: Int
+) : PanValidator {
+    override fun checkIfValid(cardNumber: String): Boolean {
+        // for example 444444 4444 4444 *** **
+        // maps to 44444444444444*****, which could be
+        // a whitelisted pattern
+        return cardNumber.startsWith(specialChar.repeat(prefixLength))
+    }
+
+    override fun checkIfComplete(cardNumber: String): Boolean {
+        return checkIfValid(cardNumber) && cardNumber.length == MAX_CARD_LENGTH
+    }
 
-    private fun setIsComplete(cardNumber: String) {
-        isComplete = passesValidation(cardNumber) &&
-            isCorrectLength(cardNumber)
+    override fun checkForValidationError(cardNumber: String): ElementValidationError? {
+        // this is a developer-only validator and should remain
+        // quiet by not returning a validationError
+        return null
     }
+}
+
+class PaymentCaptureErrorCard : WhitelistedCards("4", 14)
+class BalanceCheckErrorCard : WhitelistedCards("5", 14)
+class NonProdValidEbtCard : WhitelistedCards("9", 4)
```

Comment:
we should support [this case](https://joinforage.slack.com/archives/C056QGRF6S3/p1692290385036899) as well

Reply:
excellent callout! Will add shortly!

Reply:
Address in [7a75444bb](https://github.com/teamforage/forage-android-sdk/pull/69/commits/7a75444bb050472b3348eb784e290af37aaf412c)